docs/source/extension/extension_migration.rst

```diff
@@ -75,6 +75,12 @@ See https://github.com/jupyterlab/team-compass/issues/143 for more context on th
       ...
     ].join('|');
 
+- Some CSS rules for ``button`` with the class ``.jp-ToolbarButtonComponent`` has been kept for backward compatibility.
+
+  These rules are now **deprecated** and will be removed in Jupyterlab 5.
```

Comment:
Wondering if this removal would require a major release of JupyterLab (in that case JupyterLab 5)?

Or just a major bump of the JS package where these styles are defined (in that `@jupyterlab/ui-components`).

Reply:
At some point in the past, there were some discussions about versioning JS packages a bit more independently, without requiring a major lab release.

Reply:
Probably it's fine to leave it as is for now (mentioning JupyterLab 5). We can also discuss that separately. This just came to mind while looking at this addition to the extension migration guide.